src/config.js

```diff
@@ -10,17 +10,17 @@ import isUrl from "is-url";
 import path from "path";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
-import logging from "./logging.js";
+import logger from "./logger.js";
 
 function validateConfig(config) {
   const ajv = new Ajv2019({ allErrors: true, strict: false });
   const schema = require("../config-schema.json");
   const validateFn = ajv.compile(schema);
   const valid = validateFn(config);
   if (!valid) {
-    console.log("\nErrors:");
-    console.log(validateFn.errors);
-    console.log("");
+    logger.log("\nErrors:");
+    logger.log(JSON.stringify(validateFn.errors, null, 2));
+    logger.log("");
```

Comment:
TODO: what's the sensible output format here given we don't _necessarily_ have a valid value for `format` yet?

Reply:
Text. Always text